hlxsites/merative2
# fix(Blog article sort): Removing blogArticles sorting logic.  Offload…

…ing the logic to /blog/query-index.xlsx





## Issue

Fixes #219

## Description

Move sorting logic out of JS, and into Excel. Blog articles should be sorted by their published date in descending order. (Most recently published article should shows on top)

**Removed**

Blog article sorting logic in script/script.js

## Design Specs

- Before: https://main--merative2--hlxsites.hlx.page/blog
- After: https://219-sort-blog-home-cards-pub-date--merative2--hlxsites.hlx.page/blog
  
## Testing Instruction

Use Franklin Sidekick to preview Document > website > query-index.xlsx. Blog articles in query-index.json in each row should be ordered by their publish date in descending order.

In /blog/query-index.json, articles are sorted by their publish dates and should be in descending order (latest to oldest).

Featured articles are displayed on /blog, follow by other non-featured articles.
lastModified date on articles should not affect the sorting order in /blog/query-index.json and /blog.
